package/CHANGELOG

```diff
@@ -36,6 +36,8 @@ Changes
   * GSD is no longer a core dependency of MDAnalysis. It must be installed
     independently if you wish to read HOOMD trajectory files.
     (PR #4174, Issue #3819)
+  * Atom ID representation order updated to match that of associated AtomGroup indices.
```

Comment:
Could you:

1. Move this to the top of the list (we work by newest entry first in the changelog)
2. Keep this line to a maximum of 79 characters?